packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -112,7 +118,7 @@ const FormLabel = ({
   const width = position === 'left' && isDesktop ? '120px' : 'auto';
 
   return (
-    <Component htmlFor={htmlFor} style={{ width }} id={id}>
+    <Component htmlFor={htmlFor} style={{ width, flexShrink: 0 }} id={id}>
       <Box marginBottom="spacing.1">{textNode}</Box>
```

Comment:
Oops actually I forgot to do this: 

The margin bottom should only apply when position is top: 

```ts
position === 'left' ? 'spacing.0' : 'spacing.1'
```

Reply:
ideally shouldn't matter but fine i'll fix it